Source/WebCore/css/calc/CSSCalcTree.h

```diff
@@ -1148,6 +1169,7 @@ OP_TUPLE_LIKE_CONFORMANCE(Log, 2);
 OP_TUPLE_LIKE_CONFORMANCE(Exp, 1);
 OP_TUPLE_LIKE_CONFORMANCE(Abs, 1);
 OP_TUPLE_LIKE_CONFORMANCE(Sign, 1);
+OP_TUPLE_LIKE_CONFORMANCE(Anchor, 0);
```

Comment:
Would be good to implement this with 3, and then have a proper get() implementation, but obviously that has bunch of fallout that would need to be resolved.

The main things you are missing without this are:

- support for collectComputedStyleDependencies() (in CSSCalcTree+ComputedStyleDependencies)
- support for copy() (in CSSCalcTree+Copy)

You can manually add overloads for Anchor in the meantime.

Reply:
I tried that initially but it was just too much template-fu for this patch.

Reply:
Yeah, I need to make that easier. I have an idea I will try out when I implement random.